pkg/mgrconfig/load.go

```diff
@@ -94,12 +94,44 @@ func defaultValues() *Config {
 		Procs:          6,
 		PreserveCorpus: true,
 		Experimental: Experimental{
-			RemoteCover: true,
-			CoverEdges:  true,
+			RemoteCover:      true,
+			CoverEdges:       true,
+			DescriptionsMode: DescriptionsMode(ManualDescriptions).String(),
 		},
 	}
 }
 
+type DescriptionsMode int
+
+const (
+	Invalid = iota
```

Comment:
You can unexport it from the package interface, since nobody outside of this package is supposed to use it
Call it `invalidDescriptions`.

Reply:
Done!